_projects/2018/astropy/astropy-learn-website.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Astropy Learn Website
+desc: Design and Implement landing page for Astropy Learn ecosystem.
+# add a short one line description of your project
+requirements:
+# Student requirements:
+ - Web development
+ - User interface design and user experience
+difficulty: moderate
+issues:
+# Related issues (if any)  to this project.
+mentors:
+# First person in contact; mentors may change before project starts.
+ - Kelle Cruz
+ - Lia Corrales
+ - Adrian Price-Whelan
+ - Erik Tollerud
+initiatives:
+ - GSOC
+ - SOCIS
```

Comment:
This can be revised for the future, but as astropy we don't really do socis (esa summer of code)